Multisig/UI/App/UIKit/UI Library/ReusableViews/AddressInfoView.swift

```diff
@@ -8,18 +8,29 @@
 
 import UIKit
 
-class AddressWithTitleView: UINibView {
+struct AddressInfo {
+    let address: Address
+    let label: String?
+}
+
+class AddressInfoView: UINibView {
     @IBOutlet private weak var identiconView: UIImageView!
     @IBOutlet private weak var textLabel: UILabel!
     @IBOutlet private weak var detailLabel: UILabel!
```

Comment:
I would name this label "addressLabel" then

Reply:
ok